src/main/java/org/ossgang/commons/observable/Observables.java

```diff
@@ -31,10 +37,77 @@ private Observables() {
      * return the given value on get(). On subscribe(), it will immediately send the given value to the listener.
      *
      * @param value the value
-     * @param <T> the value type
+     * @param <T>   the value type
      * @return a constant ObservableValue
      */
     public static <T> ObservableValue<T> constant(T value) {
         return new ConstantObservableValue<>(value);
     }
+
+    /**
+     * Merges any Observables of the same type into one. Updates whenever all sources have provided at least one item.
```

Comment:
Be careful with verbs like merge/combine .... I think here it is more combining, or? MErging of streams/observables would more mean putting values one after the other ...

Reply:
Fixed in the docs